### DISCUSSION TITLE
[Modification v2] 19. Limiter le nombre de requêtes http

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#44] [Bonnes pratiques] [answered]
https://github.com/cnumr/fiches-wordpress-greenit/blob/main/best-practices-WordPress/fiches/19.%20Limiter%20le%20nombre%20de%20requ%C3%AAtes%20HTTP.md

@yannkozon, @florinesueur demande : concrètement qu’est-il possible de faire sur un Wordpress pour limiter les requêtes http ?

**Comment by florinesueur** [comment]
@dNicolle et @yannkozon, je me demande s'il y a d'autres manières que les BP déjà enoncées.

**Reply by dNicolle** [reply]
Completement, je ne pense pas que cette fiche soit complète.

**Comment by florinesueur** [comment, marked as answer]
BP en v2.
Issue : https://github.com/cnumr/fiches-wordpress-greenit/issues/95
